todoapp/src/App.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react'
+
+//=====Components=====//
+import Todos from './Todos'
+import AddTodo from './AddTodo'
+
+ class App extends Component {
+   state = {
+     todos: [
+       {id: 1, content: 'Walk the dog.'},
+       {id: 2, content: 'Play Mario Cart.'}
+     ]
+   }
+   deleteTodo = (id) => {
+     console.log(id);
+     const todos = this.state.todos.filter(todo => {
```

Comment:
good job with this filter method, remember that the fat arrow represents an implicit return. you could simple write ```const todos = this.state.todos.filter(todo => todo.id !== id);```